examples/basic/components/Button.mdx

```diff
@@ -12,28 +12,39 @@ import { Story, Stories } from 'docz-plugin-storybook/dist/react'
 <Button>Hello Button</Button>
 
 ## Button with Emoji
+
 <Button>
   <span role={'img'} aria-label={'so cool'}>
     😀 😎 👍 💯
   </span>
 </Button>
 
----
+## Styled Components Button
+
+<StyledButton>Styled Button</StyledButton>
 
-## Story with text
+---
```

Comment:
this entire `examples/basic` has been a bit of a dumping ground but at some point we should def move this type of example into its own, isolated folder so the "basic" example is as bare bones as possible.

let's ignore this for now and keep as-is, just a mental note for the future.